src/Transports/MassTransit.EventHubIntegration/Configuration/StorageSettings.cs

```diff
@@ -0,0 +1,44 @@
+namespace MassTransit.EventHubIntegration.Configuration
+{
+    using System;
+    using Azure.Core;
+    using Azure.Storage;
+    using Azure.Storage.Blobs;
+
+
+    public class StorageSettings :
```

Comment:
I would like to see all the common storage stuff be kept in MassTransit.Azure.Storage (it is currently only MessageData).